src/transaction/log_manager.c

```diff
@@ -4738,6 +4738,14 @@ log_append_commit_log_with_lock (THREAD_ENTRY * thread_p, LOG_TDES * tdes, LOG_L
 static void
 log_append_abort_log (THREAD_ENTRY * thread_p, LOG_TDES * tdes, LOG_LSA * abort_lsa)
 {
+  if (prm_get_integer_value (PRM_ID_SUPPLEMENTAL_LOG) > 0)
```

Comment:
I think you put this in `log_abort_local()` to be consistent with the commit.

Reply:
Ok, I will fix it.